src/pykeen/models/unimodal/hake.py

```diff
@@ -0,0 +1,173 @@
+# coding=utf-8
+"""Implementation of the HAKE model."""
+
+from typing import Optional
+
+import torch
+from torch import nn
+
+from pykeen.losses import Loss
+from pykeen.models import EntityRelationEmbeddingModel
+from pykeen.regularizers import Regularizer
+from pykeen.triples import TriplesFactory
+from pykeen.utils import get_embedding, get_embedding_in_long_canonical_shape
+
+
+class HAKE(EntityRelationEmbeddingModel):
+    """An implementation of HAKE [zhang2020]_.
```

Comment:
What is HAKE an acronym for?

Reply:
"Hierarchy-Aware Knowledge Graph Embeddings" -> please sneak this in to the documentation somewhere.